src/test/java/org/broadinstitute/hellbender/utils/config/ConfigUnitTest.java

```diff
@@ -0,0 +1,277 @@
+package org.broadinstitute.hellbender.utils.config;
+
+import org.aeonbits.owner.Accessible;
+import org.aeonbits.owner.ConfigCache;
+import org.aeonbits.owner.ConfigFactory;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import static java.nio.file.StandardCopyOption.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * Unit test for GATK configuration file handling.
+ *
+ * Created by jonn on 7/19/17.
+ */
+public class ConfigUnitTest extends BaseTest {
+
+    private final static String testResourceDir = publicTestDir + "org/broadinstitute/hellbender/utils/config/";
+
+    // ================================================================================
```

Comment:
These // === blocks aren't something we usually have in our code.  They can definitely be useful and I don't mind having them if it clarifies things.  It's just not something we regularly do.

We honestly don't really have any consistent policy about how to organize methods and such in test files, so coming up with one wouldn't be a bad thing.